server/routes/users.js

```diff
@@ -42,7 +72,41 @@ function checkBody(userMail, userPassword) {
          err: 'Please, provide a valid email address'
       }
    }
-   return { status: 202 }
+   return { status: 200 }
 }
 
-module.exports = { signup };
\ No newline at end of file
+async function checkUserExist(mail) {
+   const [result] = await excuteQuery('SELECT * FROM users WHERE user_mail = ?', [mail]);
+   return result;
+}
+
+async function checkUserPassword(inputPassword, dbPassword) {
+   return bcrypt.compareSync(inputPassword, dbPassword)
+}
+
+async function tokenGenerator(payload) {
+   const accessToken = jwt.sign(payload, process.env.JWT_ACCESS_SECRET_KEY, { expiresIn: process.env.JWT_ACCESS_EXPIRE });
+   return accessToken;
+}
+
+async function varifyAccessToken(request, reply) {
```

Comment:
Maybe it`s better to put this function in separate file.
This function belongs rather auth then user. It will be used by other API, for example refund